data_wrangling/inconsistent.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<p>Great job exporting the data but you should also display the result of your wrangling. Jupyter notebooks are meant for experimentation and testing, so let's see what you did and the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/gbotemi_bolarinwa/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>